OpenRA.Mods.Common/LoadScreens/BlankLoadScreen.cs

```diff
@@ -64,7 +64,7 @@ public void StartGame(Arguments args)
 			var connect = args != null ? args.GetValue("Launch.Connect", null) : null;
 			if (!string.IsNullOrEmpty(connect))
 			{
-				var parts = connect.Split(':');
+				var parts = connect.Replace("openra://", "").Split(':');
```

Comment:
Err, I just realized that I was getting mixed up with all the different `launch-*.sh`es.  The above discussion applies to a new connection script, not the replay one.